inst/workflow/metrics/kri0002.yaml

```diff
@@ -26,9 +30,10 @@ steps:
   - name: Input_Rate
```

Comment:
Let's remove strFilterNumerator and just use additional RunQuery Steps here instead